docker/compose/docker-compose.yml

```diff
@@ -96,11 +95,10 @@ services:
       ZK_URL: zookeeper:2181
       CONTROLLER_URL: tcp://${HOST_IP}:9090
       JAVA_OPTS: |
-        -Dmetrics.enableCSVReporter=false
```

Comment:
and this?

Reply:
I don't see `metrics.enableCSVReporter` in `MetricsConfig`. So, I am assuming this is unused.